### DISCUSSION TITLE
PrimeReact v10.6.4 Dropdown component issue

### DISCUSSION THREAD

**Discussion by jnlinn** [primefaces/community#1809] [PrimeReact] [answered]
I just tried to update to PrimeReact v10.6.4 and it broke all of the dropdown components in our webapp. Here is the browser error:

![browser-error](https://github.com/primefaces/community/assets/88069194/23e10a4d-a919-48c3-b680-50ff5e73ae27)

I was able to fix the dropdown components by adding the optionLabel and optionValue attributes. However, our webapp uses the Datatable component with paginator. The same error is occurring when clicking on rowsPerPageOptions dropdown. I cannot fix this.

You can see the error in your demo page: [[](url)](https://primereact.org/datatable/#paginator_basic)

![paginator-error](https://github.com/primefaces/community/assets/88069194/95a887d8-7790-4641-9881-174ca4d617df)

Please advise. Thank you.

**Comment by melloware** [comment, marked as answer]
See: https://github.com/primefaces/primereact/issues/6518
